lnbits/wallets/fake.py

```diff
@@ -80,8 +80,10 @@ async def create_invoice(
 
     async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
         invoice = decode(bolt11)
+        # TODO: no data here?
+        data: Dict = {"privkey": "missing"}
         if (
-            hasattr(invoice, "checking_id")
+            invoice.checking_id is not None
```

Comment:
Is this safe? What if invoice doesn't have the attribute "checking_id"?

Reply:
that is why i added it to invoice model, as mentioned in your review below.

Reply:
i think this piece of code is terribly broken, it was using data variable which was not defined at all.

Reply:
Do not change models (anywhere in this PR). Stick to the goals of this PR which is type annotation and safety. Please remove this attribute from the invoice.

Reply:
can you look at the code there? i have absolutely no clue why and how this piece of code work.... i think it is completely wrong.